docs/oauth2.md

```diff
@@ -0,0 +1,30 @@
+Oauth2
+======
+
+Oauth2 is a django app that will can be used to cache your `django-oauth-toolkit`
+access token model.
+
+
+Usage
+-----
+To start caching your api access tokens, add `django_toolkit.oauth2` to your
+`INSTALLED_APPS` and then add the oauth2 validator class in the `OAUTH2_PROVIDER`
+settings.
```

Comment:
It would be good to have a paragraph about the `ACCESS_TOKEN_CACHE_BACKEND` setting.